### PR TITLE
add distributed training doc

### PR DESCRIPTION
**Pull request by littletomatodonkey**
att. acceleration ratio can reach 150% when using 2 machine.

### CODE REVIEW

**Review by LDOUBLEV — APPROVED** [review]

File: doc/doc_ch/distributd_training.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+python3 -m paddle.distributed.launch \
+    --log_dir=./log/ \
+    --gpus '0,1,2,3,4,5,6,7' \
+    tools/train.py \
+    -c configs/rec/rec_mv3_none_bilstm_ctc.yml
+```
+
+### 多机训练
+
+* 相比单机训练，多机训练时，只需要添加`--ips`的参数，下面为运行代码示例。
```
Comment (MissPenguin) [inline doc/doc_ch/distributd_training.md:23]: “只需要添加`--ips`的参数”后面加一句说明吧，“该参数表示***”

File: doc/doc_ch/distributd_training.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+## 性能效果测试
+
+* 基于单机8卡P40，和2机8卡P40，在26W公开识别数据集(LSVT, RCTW, MTWI)上进行训练，最终耗时如下。
+
+|         模型             |     配置文件 |  机器数量    | 每台机器的GPU数量  |   训练时间    | 识别Acc    | 加速比 |
+| :----------------------: | :------------: | :------------: | :---------------: | :----------: | :-----------: | :-----------: |
+|          CRNN        |   configs/rec/ch_ppocr_v2.0/rec_chinese_lite_train_v2.0.yml  | 1     |  8  |  60h  |  66.7% | - |
+|          CRNN        |   configs/rec/ch_ppocr_v2.0/rec_chinese_lite_train_v2.0.yml   | 2   |  8  |  40h  |  67.0% | 75% |
+
+可以看出，精度没有下降的情况下，训练时间由60h缩短为了40h。
```
Comment (MissPenguin) [inline doc/doc_ch/distributd_training.md:51]: 补充：加速比达（计算公式）=75%

File: doc/doc_ch/distributd_training.md (hunk 1 above)
Comment (littletomatodonkey) [reply doc/doc_ch/distributd_training.md]: done

File: doc/doc_ch/distributd_training.md (hunk 2 above)
Comment (littletomatodonkey) [reply doc/doc_ch/distributd_training.md]: done

**Review by MissPenguin — APPROVED** [review]

**Pull request merged**